examples/nginx-access-logs-streaming/README.md

```diff
@@ -0,0 +1,640 @@
+│ 117.132.188.xxx │        100 │           73709.66 │ China          │ 🇨🇳    │
+│ 112.175.187.xxx │         11 │  69961.81818181818 │ South Korea    │ 🇰🇷    │
+│ 106.38.221.xxx  │         27 │  69016.77777777778 │ China          │ 🇨🇳    │
+│ 66.249.75.xxx   │         44 │  63793.52272727273 │ United States  │ 🇺🇸    │
+│ 102.91.4.xxx    │         23 │  60814.13043478261 │ Nigeria        │ 🇳🇬    │
+│ 62.250.221.xxx  │         14 │  57457.57142857143 │ Netherlands    │ 🇳🇱    │
+│ 182.69.182.xxx  │         24 │           57162.25 │ India          │ 🇮🇳    │
+│ 71.56.25.xxx    │         21 │ 55815.666666666664 │ United States  │ 🇺🇸    │
+│ 209.122.69.xxx  │         15 │            51437.8 │ United States  │ 🇺🇸    │
+│ 66.253.176.xxx  │         15 │            51437.8 │ United States  │ 🇺🇸    │
+│ 172.98.32.xxx   │         15 │  51437.73333333333 │ United States  │ 🇺🇸    │
+│ 185.119.0.xxx   │         78 │  50131.61538461538 │ Russia         │ 🇷🇺    │
+│ 85.76.79.xxx    │         14 │ 49400.357142857145 │ Finland        │ 🇫🇮    │
+│ 49.229.186.xxx  │         14 │  49400.28571428572 │ Thailand       │ 🇹🇭    │
+│ 172.59.189.xxx  │         14 │  49400.28571428572 │ United States  │ 🇺🇸    │
+│ 107.214.5.xxx   │         14 │  49400.28571428572 │ United States  │ 🇺🇸    │
+│ 72.227.218.xxx  │         14 │  49400.28571428572 │ United States  │ 🇺🇸    │
+│ 118.92.17.xxx   │         14 │  49400.28571428572 │ New Zealand    │ 🇳🇿    │
+│ 44.232.17.xxx   │         13 │  46098.53846153846 │ United States  │ 🇺🇸    │
+│ 50.237.143.xxx  │         14 │ 45253.142857142855 │ United States  │ 🇺🇸    │
+│ 96.230.1.xxx    │         14 │  45253.07142857143 │ United States  │ 🇺🇸    │
+│ 167.172.52.xxx  │         25 │           44107.76 │ United Kingdom │ 🇬🇧    │
+│ 94.130.16.xxx   │        334 │   44107.6377245509 │ Germany        │ 🇩🇪    │
+│ 87.250.179.xxx  │        378 │ 44107.589947089946 │ United States  │ 🇺🇸    │
+│ 65.109.142.xxx  │         84 │  44107.55952380953 │ Finland        │ 🇫🇮    │
+│ 8.29.198.xxx    │         15 │  44107.53333333333 │ United States  │ 🇺🇸    │
+│ 104.254.65.xxx  │         18 │            44107.5 │ United States  │ 🇺🇸    │
+│ 216.18.205.xxx  │         30 │ 44107.433333333334 │ United States  │ 🇺🇸    │
+│ 65.109.156.xxx  │         94 │  44107.40425531915 │ Finland        │ 🇫🇮    │
+│ 86.155.139.xxx  │         13 │              44104 │ United Kingdom │ 🇬🇧    │
+│ 47.242.141.xxx  │        152 │  44056.89473684211 │ Hong Kong      │ 🇭🇰    │
+│ 45.11.60.xxx    │        137 │  44056.24087591241 │ Poland         │ 🇵🇱    │
+│ 84.140.146.xxx  │         19 │  44056.15789473684 │ Germany        │ 🇩🇪    │
+│ 71.62.20.xxx    │         16 │              44056 │ United States  │ 🇺🇸    │
+└─────────────────┴────────────┴────────────────────┴────────────────┴───────┘
+
+50 rows in set. Elapsed: 0.068 sec. Processed 68.29 thousand rows, 642.63 KB (1.01 million rows/s., 9.51 MB/s.)
+```
+
```

Comment:
Please add a Summary section.

It will be even better if you can use Grafana Proton plugin to create dashboard/charts, before the summary section. But totally optional for sure.

Reply:
Added a summary as requested.

This post is already quite dense as is. 

I think Grafana plugin + Proton (Cloud) can continue from where this left off in a separate post.